.gitlab/packaging/deb.yml

```diff
@@ -12,6 +12,7 @@
     - echo "Packaging artifact ${OMNIBUS_PACKAGE_ARTIFACT} to .deb"
     - !reference [.setup_deb_signing_key]
     - inv -e omnibus.build --release-version "$RELEASE_VERSION" --base-dir $OMNIBUS_BASE_DIR --skip-deps --target-project installer
```

Comment:
@chouquette noticed this seems to be the only packaging job that doesn't `ls` the `OMNIBUS_BASE_DIR` - not sure if you/we care, but figured I'd mention.

Reply:
it's a nice to have, but I don't think it matters much as it's only for debug/control purposes